library/build.gradle

```diff
@@ -21,6 +21,14 @@ android {
     defaultConfig {
         minSdkVersion 14
         targetSdkVersion 22
+
+        def projects = rootProject.getProject().getChildProjects()
+        projects.each {
+            def authority = it.value.android.defaultConfig.manifestPlaceholders.downloadAuthority
```

Comment:
I wonder if `?.` instead of just `.` might make that more robust when having additional modules.

Also `rootProject.subProjects` returns a set of projects so that might avoid the `it.value`, `it` would be the project then already. Or do you rely on the map of projects for some reason?